api/users.js

```diff
@@ -0,0 +1,92 @@
+var express = require('express');
+var router = express.Router();
+const db = require('../db/db');
+const jwt = require('jsonwebtoken')
+const bcrypt = require('bcrypt'); // pass ne smije biti NUL
+
+process.env.SECRET_KEY = 'secret'
```

Comment:
This should not be initialized here, but in .env file which should not be pushed to git.